src/Tizen.Network.Stc/Tizen.Network.Stc/StcRule.cs

```diff
@@ -0,0 +1,288 @@
+                }
+                return from;
+            }
+        }
+
+        /// <summary>
+        /// A property to get "to" value(end) of time interval for statistics rule.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>to(end) of time interval.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public DateTime To
+        {
+            get
+            {
+                DateTime from;
+                DateTime to;
+                int ret = Interop.Stc.Rule.GetTimeInterval(_ruleHandle, out from, out to);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get time interval(to value) for rule, Error - " + (StcError)ret);
+                }
+                return to;
+            }
+        }
+
+        /// <summary>
+        /// To get Interface type for statistics rule.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <returns>Interface type.</returns>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public NetworkInterface GetInterfaceType()
+        {
+            NativeNetworkInterface ifaceType;
+            int ret = Interop.Stc.Rule.GetInterfaceType(_ruleHandle, out ifaceType);
+            if (ret != (int)StcError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to get Interface type, Error - " + (StcError)ret);
+            }
+            if (ifaceType == NativeNetworkInterface.Unknown)
+            {
+                throw new InvalidOperationException("Interface Type is Unknown.");
+            }
+            return (NetworkInterface)ifaceType;
+        }
+
+        /// <summary>
+        /// To set Interface type for statistics rule.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <param name="interfaceType"> Interface type.</param>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public void SetInterfaceType(NetworkInterface interfaceType)
```

Comment:
Which value should I set to select any interface type?

Reply:
'Wifi' can be selected as interface type.

Reply:
Why not `NetworkInterface.All` ?

Reply:
Sorry, I misunderstood earlier. Actually I tried setting NetworkInterface.All (as the interface type of Rule) in Core APIs' testapp, but then it does not return any data. When I set the NetworkInterface to 'Wifi' or 'Unknown(Default value)', it gives some statistics of apps accordingly.